styles/globals.css

```
@@ -55,4 +54,63 @@ body {
 
 .right-navbar-container a {
   @apply text-violet-700 text-xl;
-}
\ No newline at end of file
+}
+
+/* ========== Sidebar Component ========== */
+
+.top-sidebar-container span,
+h3 {
+  @apply text-white;
+}
+
+.top-sidebar-container :first-child {
+  @apply text-2xl font-semibold;
+}
+
+.top-sidebar-container {
+  @apply pt-12 pl-12 flex flex-col gap-y-2;
+}
+
+.side-menu {
+  @apply flex gap-x-3 pt-4 pb-4 pl-6 text-2xl;
+}
+
+.sidemenu-border {
+  @apply pt-6 w-1/2 border-b-2 border-zinc-100;
+}
+
```

จากที่แก้ระยะของ running,เส้น,newpost ให้เท่ากัน แต่เส้นมันจะติดขอบซ้าย ต้องมาเขียนตามบรรทัดข้างล้างเพิ่ม แต่เส้นมันจะอยู่ตรงกลาง ก็ได้อยู่ ถ้าอยากปรับเพิ่มก็ลองปรับดูได้
```suggestion

.sidemenu-border {
  @apply pt-5 w-1/2 border-b-2 border-zinc-100 self-center;
}

.sidebar-container {
  @apply flex flex-col;
}

```